@AlexanderRichert-NOAA - I guess the new logic in bacio isn't working as expected. I'll go back to what we had in our spack fork until there is a proper fix in the authoritative repo?
```
    '/usr/local/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/home/runner/work/spack-stack/spack-stack/envs/default/install/intel/2021.8.0/bacio-2.4.1-pjld7qk' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DBUILD_TESTING:BOOL=OFF' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/home/runner/work/spack-stack/spack-stack/envs/default/install/intel/2021.8.0/bacio-2.4.1-pjld7qk/lib;/home/runner/work/spack-stack/spack-stack/envs/default/install/intel/2021.8.0/bacio-2.4.1-pjld7qk/lib64' '-DCMAKE_PREFIX_PATH:STRING=' '-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON' '/home/runner/work/spack-stack/spack-stack/cache/build_stage/spack-stage-bacio-2.4.1-pjld7qkpid7oxovqaxhbi7auxsip6gd5/spack-src'

1 error found in build log:
  >> 3    CMake Error at CMakeLists.txt:5 (project):
     4      VERSION "2.4.12.4.1" format invalid.
     5    
     6    
     7    -- Configuring incomplete, errors occurred!
```

--- a/configs/common/packages.yaml
+++ b/configs/common/packages.yaml
@@ -216,13 +216,13 @@
       version: [3.1.0]
     py-python-dateutil:
       version: [2.8.2]
-    py-pythran:
-      # Earlier versions don't compile on macOS with llvm-clang/13.0.0 and Python/3.9
-      version: [0.11.0]
+    #py-pythran:
+    #  # Earlier versions don't compile on macOS with llvm-clang/13.0.0 and Python/3.9
+    #  version: [0.11.0]
     py-pyyaml:
       version: [6.0]
-    py-scipy:
-      version: [1.8.0]
+    #py-scipy:
+    #  version: [1.8.0]
     # Pin the py-setuptools version to avoid duplicate Python packages
     py-setuptools:
       version: [59.4.0]

